docs/help.md

```diff
@@ -0,0 +1,123 @@
+# Help
+
+* [Log rotation](#rotate)
+* [Saving to multiple files](#multiple)
+* [Log Filtering](#filter-logs)
+* [Transports and systemd](#transport-systemd)
+* [Duplicate keys](#dupe-keys)
+* [Log levels as labels instead of numbers](#level-string)
+* [Pino with `debug`](#debug)
+
+<a id="rotate"></a>
+## Log rotation
```

Comment:
Should we have a link in here somewhere to https://www.nearform.com/blog/sematext-guest-post-pino-fastest-node-js-logger-production/ with a specific mention of using `logagent`?

Reply:
we don't link out to blog posts in general.. I think that will become a burden to maintain over time